transcript-client/src/utils/draftJsStylingUtils.ts

```diff
@@ -1,6 +1,5 @@
 // src/utils/draftJsStylingUtils.ts
-//@ts-ignore
```

Comment:
The reason why we added this is because there is no ts for draft js. So I don't why you remove this

Reply:
Run `npm install`. I added the typescripts definition package for `draft-js` so we don't have to use `//@ts-ignore`:

```json
  "devDependencies": {
    "@types/draft-js": "^0.11.17",
    "eslint": "^8.56.0"
  }
```

I do not know why there are so many `//@ts-ignore`s in the frontend codebase. Why even use typescript if you are just going to override and ignore the compiler warnings?

Reply:
It is not about us ignoring the compiler warning the draft js ts is not supported at the time we were doing this. Either way the changes in frontend are causing issue and everything works perfect before the changes. I will just add the types/draft-js